sql/core/src/test/resources/sql-tests/results/ansi/interval.sql.out

```diff
@@ -1136,117 +1136,157 @@ struct<INTERVAL '-178956970-8' YEAR TO MONTH:interval year to month>
 
 
 -- !query
-create temporary view interval_arithmetic as
-  select CAST(dateval AS date), CAST(tsval AS timestamp), dateval as strval from values
-    ('2012-01-01', '2012-01-01')
-    as interval_arithmetic(dateval, tsval)
+select
+  interval '2-2' year to month + interval '3' month,
+  interval '2' year - interval '3-3' year to month,
+  interval '99 11:22:33.123456789' day to second + interval '10 9:8' day to minute,
+  interval '22:33.123456789' minute to second - interval '10' day
+-- !query schema
+struct<(INTERVAL '2-2' YEAR TO MONTH + INTERVAL '3' MONTH):interval year to month,(INTERVAL '2' YEAR - INTERVAL '3-3' YEAR TO MONTH):interval year to month,(INTERVAL '99 11:22:33.123456' DAY TO SECOND + INTERVAL '10 09:08' DAY TO MINUTE):interval day to second,(INTERVAL '22:33.123456' MINUTE TO SECOND - INTERVAL '10' DAY):interval day to second>
+-- !query output
+2-5	-1-3	109 20:30:33.123456000	-9 23:37:26.876544000
+
+
+-- !query
+select
+  interval '2' year + '3-3 year to month',
+  interval '3 year' - interval '2-2' year to month,
+  interval '99 11:22:33.123456789' day to second + '12:12 hour to second',
+  interval '4 22 day to hour' - interval '10' day,
+  interval '2' year + null,
+  null - interval '4 22' day to hour
 -- !query schema
 struct<>
 -- !query output
+org.apache.spark.sql.catalyst.parser.ParseException
+
+Cannot parse the INTERVAL value: 4 22 day to hour(line 5, pos 2)
```

Comment:
I think this is a bug, cc @AngersZhuuuu @MaxGekk

Reply:
> I think this is a bug, cc @AngersZhuuuu @MaxGekk

Got it, will raise a pr and work on this.